HelpDeskCommonMethods/src/main/java/webdriverbase/AppPage.java

```diff
@@ -31,7 +31,7 @@ public class AppPage {
 	public static String WINDOWS_PATH_TO_TEST_DATA_DIR = "src/main/resources/";
 	public static int WAIT_TIME_SEC = 60;	
 	protected WebDriver driver ;
-	
+	//
```

Comment:
On Line 104, the tester has mentioned Thread.sleep. It is generally bad practice to use Thread.sleep(). Sleeps aren't dynamic. They only wait for the specified time... no more, no less. This generally is not good because if the element you are waiting for comes back in 25ms, then you will wait the full 10s. If the element comes at 10.5s, then it will fail.